stream_alert_cli/config.py

```diff
@@ -18,17 +18,19 @@
 import os
 import sys
 
+from collections import OrderedDict
+
 from stream_alert_cli.logger import LOGGER_CLI
 
 
 class CLIConfig(object):
     '''Provide an object to load, modify, and display the StreamAlertCLI Config'''
 
     def __init__(self):
-        self.config_files = {
+        self.config_files = OrderedDict({
```

Comment:
I don't think you're actually using is as an OrderedDict now (so this could be removed) but I also thought I'd point out something I noticed a while back. This is a weird thing with OrderedDict - but you can't cast a dict to OrdererDict like this and have it maintain order (which I guess makes sense). For instance, if you run this code:

```
from collections import OrderedDict

test_dict = OrderedDict({
    'global2': 'conf/global.json',
    'global1': 'conf/global.json',
    'lambda1': 'conf/lambda.json',
    'lambda2': 'conf/lambda.json',
    'lambda3': 'conf/lambda.json',
    'lambda4': 'conf/lambda.json',
    'lambda5': 'conf/lambda.json'
})

for key, value in test_dict.iteritems():
    print key, value
```
It will print not print in the order you expect. It will probably look _similar_ to:
```
global1 conf/global.json
global2 conf/global.json
lambda4 conf/lambda.json
lambda5 conf/lambda.json
lambda1 conf/lambda.json
lambda2 conf/lambda.json
lambda3 conf/lambda.json
```

The actual not-so-obvious way of doing this is through casting a list of ordered tuples:
```
OrderedDict([
    ('global', 'conf/global.json'),
    ('lambda', 'conf/lambda.json')
])
```

I think you just want to remove the cast to OrderedDict until we figure out how to better test this stuff.